test/css/test-ruby.css

```diff
@@ -0,0 +1,110 @@
+@charset "UTF-8";
+/* # Module: Settings */
+/* ------------------ */
+/* Test: true-context [mixin] */
+/*   ✔ Confirm that there is currently no "fake" scope */
```

Comment:
@mirisuzanne what problem did they cause? I have a vague memory of discussing that, but I thought the conclusion was that in fact they were fine.